app/views/waste_exemptions_engine/renew_without_changes_forms/new.html.erb

```diff
@@ -0,0 +1,14 @@
+<%= render("waste_exemptions_engine/shared/back", back_path: back_renew_without_changes_forms_path(@renew_without_changes_form.token)) %>
+
+<div class="text">
+  <%= form_for(@renew_without_changes_form) do |f| %>
+    <%= render("waste_exemptions_engine/shared/errors", object: @renew_without_changes_form) %>
+
+    <h1 class="heading-large"><%= t(".heading") %></h1>
+
```

Comment:
Are you adding the content for this in a separate PR?

Reply:
Yes :)